packages/fuels/tests/harness.rs

```diff
@@ -3708,3 +3670,91 @@ async fn test_rust_result_can_be_encoded() -> Result<(), Box<dyn std::error::Err
 
     Ok(())
 }
+
+#[tokio::test]
```

Comment:
What do you think about temporarily adding the flag into our CI and uncommenting these tests?

But then again SDK devs would need to remember to compile the sway projects with the flag when developing. Perhaps if you add it to the 'rust-build-sway-projects` script and in the readme as well? We can remove it as soon as this is not hidden behind a feature flag.

Reply:
Not opposed to it if the team agrees.

Reply:
:+1: